keps/sig-release/3344-refine-release-team/README.md

```diff
@@ -0,0 +1,894 @@
+updates.
+
+[documentation style guide]: https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md
+-->
+
+This change will introduce a "Release Team Roster".
+This roster will be a list of contributors, maintained by SIG Release, who wish to contribute to current or future release teams.
+
+Release teams will largely keep their existing structure, with some naming updates, including the distinct subteams and functions.
+Release teams leads (the release lead, and the subteam leads) will be selected via nomination in a process that is largely unchanged.
+The rest of the team will be comprised of roster members and shadows. 
+
+## Motivation
+
+Each Kubernetes release is driven (from a procedure perspective) by a [release team](https://github.com/kubernetes/sig-release/tree/master/release-team) of 25–30 people.
+This team works for the entire cycle of the release — approximately 16 weeks.
+
+Presently each team is selected by nomination of established community members (typically release team members in the previous release) and a shadow survey, depending on the role within the team.
+The community has identified a number of issues that arise from this procedure:
+
+- Having served on a release team is no guarantee of selection in the next team.
+  There are many cases of contributors serving for one or multiple releases, and then simply not being selected for future releases.
+  There is little feedback, and little the contributor can do other than re-apply next time.
+  This process can be stressful for contributors.
+- The release team is not "stable", as continuing requires constant re-application through the shadow survey.
+  This inhibits the ability of contributors to gain comfort and skill in a particular role. 
+- Wider community members interact with a constantly changing team of people for every release, which makes it harder for them to build working relationships with the release team.
+- With a few notable exceptions, once a contributor ends their time on the release team they don't come back.
+  This "up or out" process makes it hard to do release team work as a long-standing community role.
+
+There are a number of qualities to the current process that are desirable, and should be kept:
+
+- The shadow survey is an excellent "on ramp" into the community, providing contributors with no experience a path to [org membership](https://github.com/kubernetes/community/blob/master/community-membership.md) and it is expected that contributors may only join the release team for a release or two before moving to other parts of the community.
+- The release team having some degree of 'churn' is natural and expected.
+  An entirely static, or effectively static, team is not desirable. 
+
+Balancing these desires, this KEP introduces a long-lived "roster" of release team members.
+This roster is used to assemble release teams instead of a new shadow survey each time.
```

Comment:
I think we should not replace the shadow survey for new folks who wanna join, so let's clarify:
```suggestion
This roster is used to assemble release teams with recurring contributors in addition to the shadow survey for new folks.
```